gptel-context.el

```diff
@@ -84,6 +84,23 @@ context chunk.  This is accessible as, for example:
   :group 'gptel
   :type 'function)
 
+(defun gptel-context-add-current-kill (&optional arg)
+  "Add current-kill to gptel, accumulating if arg is non-nil"
+  (interactive "P")
+  (let ((kill (current-kill 0)))
+    (with-current-buffer (get-buffer-create "*current-kill*")
```

Comment:
Could we rename this to `*gptel-current-kill*` or `*gptel-kill-ring-context*`, something like that?  It should be easy to identify the provenance of this buffer in the buffer list.

On another note, we don't need undo enabled in this buffer and it can be a hidden buffer, so the name should start with a space, like ` *gptel-current-kill*`.